karma.conf.js

```diff
@@ -135,6 +136,8 @@ const jasmineConfig = {
           functions: 50,
           lines: 80,
           excludes: [
+            'adapter.ts',
```

Comment:
Just to verify: this excludes the adapter and constants from coverage checks? (which seems good to me since they're not effectively testable)

Reply:
Yes, this mirrors the original coverage checks for Mocha tests: https://github.com/material-components/material-components-web/blob/50f110a6cf8100e594bdbd6c02ee278c39924008/karma.conf.js#L70

Reply:
💯